src/Serilog.Sinks.OpenTelemetry/OpenTelemetryLoggerConfigurationExtensions.cs

```diff
@@ -33,23 +34,30 @@ public static class OpenTelemetryLoggerConfigurationExtensions
 #else
         null;
 #endif
-    
+
     /// <summary>
     /// Send log events to an OTLP exporter.
     /// </summary>
     /// <param name="loggerSinkConfiguration">
     /// The `WriteTo` configuration object.
     /// </param>
     /// <param name="configure">The configuration callback.</param>
+    /// <param name="includeEnvironment">If true the configuration will be overridden with values from OTLP Exporter Configuration environment variables(https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/).</param>
```

Comment:
```suggestion
    /// <param name="includeEnvironment">If true the configuration will be overridden with values from <see href="https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/">OTLP Exporter Configuration environment variables</a>.</param>
```

Reply:
I've added this suggestion, but seems that Visual Studio doesn't show the link for param, in summary this link works fine, but in param it shows as regular text.